Code/Editor/Viewport.cpp

```diff
@@ -398,11 +389,8 @@ void QtViewport::Update()
     m_viewportUi.Update();
 
     m_bAdvancedSelectMode = false;
-    bool bSpaceClick = false;
-    {
-        bSpaceClick = CheckVirtualKey(Qt::Key_Space) & !CheckVirtualKey(Qt::Key_Shift) /*& !CheckVirtualKey(Qt::Key_Control)*/;
-    }
-    if (bSpaceClick && hasFocus())
+
+    if (CheckVirtualKey(Qt::Key_Space) && !CheckVirtualKey(Qt::Key_Shift) && hasFocus())
```

Comment:
Any idea if this and `QtViewport::HitTest` can go away any time soon? I know I've seen this and considered pulling it out before, I think this is one of the last bits of hardcoded input handling in the viewport.

Reply:
That's a good point, it most likely can and I can try that and see what else complains in a follow-up PR 🙂 Thanks!